src/main/java/com/theironyard/entities/Review.java

```diff
@@ -0,0 +1,77 @@
+package com.theironyard.entities;
+
+import javax.persistence.*;
+
+/**
+ * Created by halleyfroeb on 10/14/16.
+ */
+@Entity
+@Table(name = "reviews")
+public class Review {
+    @Id
+    @GeneratedValue
+    int id;
+
+    @Column(nullable = false)
+    String author;
+
+    @Column(nullable = false)
+    String text;
+
+    @Column(nullable = false)
+    int lecturerId;
```

Comment:
FYI, If the review is tied to the Lecturer shouldn't you be using the @ManyToOne annotation with a Lecture type field? Luckily, the UI does not call an individual Lecturer's reviews so this project does not have to have them linked.